src/ssl.c

```diff
@@ -11461,21 +11469,28 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
 
     long wolfSSL_CTX_get_options(WOLFSSL_CTX* ctx)
     {
-        (void)ctx;
         WOLFSSL_ENTER("wolfSSL_CTX_get_options");
         WOLFSSL_MSG("wolfSSL options are set through API calls and macros");
-
-        return 0;
+        return ctx->mask;
```

Comment:
We should check ctx for NULL first, to prevent possible seg fault.

Reply:
Done.